journal/modernized/08-chapter-5.adoc

```diff
@@ -430,7 +430,7 @@ from the first to the last.
 In a short time a messenger came to say that the Prince was at liberty;
 when we were ushered into his apartment of ample size.
 The Prince came forward, and met us in a very courteous manner;
-and on +++_______+++ introducing me, he took me by the hand,
+and on my companion introducing me, he took me by the hand,
```

Comment:
There are several places in these journals where a name has intentionally been left out, and been replaced with a group of underscores _______ (I try to always use **seven** underscores to be consistent). Sometimes I put the name back in if I know for sure who it is referring to. But often the name was left out because it is saying something personal or negative about them, or because they didn't end well, etc. If we don't know for sure who it is referring to, we just leave the underscores there. And they always have to be wrapped in three pluses, like this +++_______+++